code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -896,25 +896,25 @@ Code:
 		return
 
 /datum/computer/file/text/bomb_manual
-	name = "Bomb Readme"
+	name = "Self-destruct Readme"
```

Comment:
This manual covers both the missile and self destruct programs, so probably should be named as such

Reply:
Understood